parsers/txt2tags.c

```diff
@@ -0,0 +1,195 @@
+/*
+*   Copyright (c) 2009, Eric Forgeot
+*   Copyright (c) 2014, Colomban Wendling <colomban@geany.org>
+*
+*   Based on work by Jon Strait
+*
+*   This source code is released for free distribution under the terms of the
+*   GNU General Public License version 2 or (at your opinion) any later version.
+*
+*   This module contains functions for generating tags for Txt2tags files
+*   (https://en.wikipedia.org/wiki/Txt2tags).
+*/
+
+/*
+*   INCLUDE FILES
+*/
+#include "general.h"	/* must always come first */
+
+#include <ctype.h>
+#include <string.h>
+
+#include "parse.h"
+#include "read.h"
+#include "nestlevel.h"
+#include "vstring.h"
+#include "routines.h"
+#include "entry.h"
+
+
+/* as any character may happen in an input, use something highly unlikely */
+#define SCOPE_SEPARATOR "\x3" /* ASCII ETX */
```

Comment:
I would like to change this to `""`. Is this acceptable?

We have used `""` in the parser for markdown and tex.
I would like to use `""` as seperators for languages for documentation.

https://github.com/universal-ctags/ctags/issues/2434

Reply:
No problem for Geany, I'm just wondering, if `""` is a good choice as it isn't sufficiently unlikely to appear in text IMO. It's an empty string in programming languages so it could appear in texts about programming languages.

Reply:
You are right. It isn't an ideal choice. I could not find such a better choice when working on markdown and tex.
However, the consistency between parsers dealing with languages for documents may make developing a client tool a bit easier.

So could you change the separator to `""`?

After merging this, I will introduce a definition:
```
#define DEFAULT_DOCLANG_SCOPE_SEPARATOR "\"\""
```

We have the following option as the last resort:
```
  --_scopesep-<LANG>=[parent_kind_letter]/child_kind_letter:separator
       Specify scope separator between <PARENT_KIND> and <KIND>.
       * as a kind letter matches any kind.
```

Reply:
Done.

I've also updated other-projects.rst to remove some outdated information.

Reply:
Thank you.